[2025-03-10T14:38:42.208+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-10T14:38:42.232+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: xkcd_daily.dbt_run manual__2025-03-10T14:20:39.848994+00:00 [queued]>
[2025-03-10T14:38:42.238+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: xkcd_daily.dbt_run manual__2025-03-10T14:20:39.848994+00:00 [queued]>
[2025-03-10T14:38:42.238+0000] {taskinstance.py:2867} INFO - Starting attempt 9 of 10
[2025-03-10T14:38:42.248+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): dbt_run> on 2025-03-10 14:20:39.848994+00:00
[2025-03-10T14:38:42.252+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=591) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-03-10T14:38:42.252+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'xkcd_daily', 'dbt_run', 'manual__2025-03-10T14:20:39.848994+00:00', '--job-id', '156', '--raw', '--subdir', 'DAGS_FOLDER/XKCD_daily/dag.py', '--cfg-path', '/tmp/tmpy0_s_0f4']
[2025-03-10T14:38:42.254+0000] {standard_task_runner.py:72} INFO - Started process 592 to run task
[2025-03-10T14:38:42.254+0000] {standard_task_runner.py:105} INFO - Job 156: Subtask dbt_run
[2025-03-10T14:38:42.277+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:209 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-03-10T14:38:42.317+0000] {task_command.py:467} INFO - Running <TaskInstance: xkcd_daily.dbt_run manual__2025-03-10T14:20:39.848994+00:00 [running]> on host 157a64d0872f
[2025-03-10T14:38:42.411+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='xkcd_daily' AIRFLOW_CTX_TASK_ID='dbt_run' AIRFLOW_CTX_EXECUTION_DATE='2025-03-10T14:20:39.848994+00:00' AIRFLOW_CTX_TRY_NUMBER='9' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-10T14:20:39.848994+00:00'
[2025-03-10T14:38:42.411+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-10T14:38:42.418+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-03-10T14:38:42.424+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'cd /dbt/transformations && dbt run --project-dir /dbt/transformations --profiles-dir /dbt']
[2025-03-10T14:38:42.429+0000] {subprocess.py:99} INFO - Output:
[2025-03-10T14:38:43.625+0000] {subprocess.py:106} INFO - [0m14:38:43  Running with dbt=1.9.3
[2025-03-10T14:38:43.782+0000] {subprocess.py:106} INFO - [0m14:38:43  Registered adapter: postgres=1.9.0
[2025-03-10T14:38:44.177+0000] {subprocess.py:106} INFO - [0m14:38:44  [[33mWARNING[0m]: Configuration paths exist in your dbt_project.yml file which do not apply to any resources.
[2025-03-10T14:38:44.178+0000] {subprocess.py:106} INFO - There are 1 unused configuration paths:
[2025-03-10T14:38:44.178+0000] {subprocess.py:106} INFO - - models.transformations.example
[2025-03-10T14:38:44.294+0000] {subprocess.py:106} INFO - [0m14:38:44  Found 3 models, 1 source, 433 macros
[2025-03-10T14:38:44.298+0000] {subprocess.py:106} INFO - [0m14:38:44
[2025-03-10T14:38:44.299+0000] {subprocess.py:106} INFO - [0m14:38:44  Concurrency: 1 threads (target='prod')
[2025-03-10T14:38:44.303+0000] {subprocess.py:106} INFO - [0m14:38:44
[2025-03-10T14:38:44.534+0000] {subprocess.py:106} INFO - [0m14:38:44  1 of 3 START sql table model dbt_dwh.dim_comics ................................ [RUN]
[2025-03-10T14:38:45.215+0000] {subprocess.py:106} INFO - [0m14:38:45  1 of 3 OK created sql table model dbt_dwh.dim_comics ........................... [[32mSELECT 561[0m in 0.68s]
[2025-03-10T14:38:45.217+0000] {subprocess.py:106} INFO - [0m14:38:45  2 of 3 START sql table model dbt_dwh.fact_cost ................................. [RUN]
[2025-03-10T14:38:45.286+0000] {subprocess.py:106} INFO - [0m14:38:45  2 of 3 OK created sql table model dbt_dwh.fact_cost ............................ [[32mSELECT 561[0m in 0.07s]
[2025-03-10T14:38:45.290+0000] {subprocess.py:106} INFO - [0m14:38:45  3 of 3 START sql table model dbt_dwh.fact_review ............................... [RUN]
[2025-03-10T14:38:45.605+0000] {subprocess.py:106} INFO - [0m14:38:45  3 of 3 OK created sql table model dbt_dwh.fact_review .......................... [[32mSELECT 561[0m in 0.31s]
[2025-03-10T14:38:45.631+0000] {subprocess.py:106} INFO - [0m14:38:45
[2025-03-10T14:38:45.632+0000] {subprocess.py:106} INFO - [0m14:38:45  Finished running 3 table models in 0 hours 0 minutes and 1.33 seconds (1.33s).
[2025-03-10T14:38:45.679+0000] {subprocess.py:106} INFO - [0m14:38:45
[2025-03-10T14:38:45.681+0000] {subprocess.py:106} INFO - [0m14:38:45  [32mCompleted successfully[0m
[2025-03-10T14:38:45.682+0000] {subprocess.py:106} INFO - [0m14:38:45
[2025-03-10T14:38:45.682+0000] {subprocess.py:106} INFO - [0m14:38:45  Done. PASS=3 WARN=0 ERROR=0 SKIP=0 TOTAL=3
[2025-03-10T14:38:46.578+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-03-10T14:38:46.623+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-10T14:38:46.624+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=xkcd_daily, task_id=dbt_run, run_id=manual__2025-03-10T14:20:39.848994+00:00, execution_date=20250310T142039, start_date=20250310T143842, end_date=20250310T143846
[2025-03-10T14:38:46.681+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-10T14:38:46.702+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-03-10T14:38:46.707+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
